prompt_toolkit/completion/fuzzy_completer.py

```diff
@@ -65,45 +96,76 @@ def sort_key(fuzzy_match):
         fuzzy_matches = sorted(fuzzy_matches, key=sort_key)
```

Comment:
Based on this I'm assuming that this completer wasn't designed to emit the completions as they come in. But even so it's still concerning that this blocks. I have `complete_in_thread` set to True.